ldes-fragmentisers/ldes-fragmentisers-common/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentation/FragmentationStrategyCreatorImplTest.java

```diff
@@ -46,43 +47,26 @@ void setUp() {
 	@Test
 	void when_ViewSpecificationFragmentationConfigIsNull_FragmentationStrategyImplIsReturned() {
 		ViewSpecification viewSpecification = new ViewSpecification(VIEW_NAME, List.of(), List.of(), 100);
-		FragmentationStrategyWrapper paginationFragmentationStrategyWrapper = Mockito
-				.mock(FragmentationStrategyWrapper.class);
-		when(applicationContext.getBean(PAGINATION_FRAGMENTATION)).thenReturn(paginationFragmentationStrategyWrapper);
-		FragmentationStrategy paginationFragmentationStrategy = Mockito.mock(FragmentationStrategy.class);
-		when(paginationFragmentationStrategyWrapper.wrapFragmentationStrategy(eq(applicationContext),
-				any(FragmentationStrategyImpl.class),
-				eq(viewSpecification.getPaginationProperties())))
-				.thenReturn(paginationFragmentationStrategy);
 
 		FragmentationStrategy fragmentationStrategy = fragmentationStrategyCreator
 				.createFragmentationStrategyForView(viewSpecification);
 
-		assertEquals(paginationFragmentationStrategy, fragmentationStrategy);
+		assertThat(fragmentationStrategy).isOfAnyClassIn(FragmentationStrategyImpl.class);
```

Comment:
As you are only checking for one class type, I would suggest to use `isInstanceOf()` instead of `isOfAnyClassIn()`